app/controllers/admin/calendar_controller.rb

```diff
@@ -1,5 +1,25 @@
 class Admin::CalendarController < Admin::AdminController
   def index
     skip_authorization
+
+    @division = current_division
+    @loans = current_division.loans
+    @events = []
+
+    @loans.each do |loan|
+      self.prepare_event(loan.calendar_start_event)
```

Comment:
In any Ruby class. self is implicit in most cases.

On Fri, Apr 15, 2016 at 1:47 PM, Elizabeth Mitchell <
notifications@github.com> wrote:

> In app/controllers/admin/calendar_controller.rb
> <https://github.com/sassafrastech/madeline_system/pull/64#discussion_r59912789>
> :
>
> > @@ -1,5 +1,25 @@
> >  class Admin::CalendarController < Admin::AdminController
> >    def index
> >      skip_authorization
> > +
> > +    @division = current_division
> > +    @loans = current_division.loans
> > +    @events = []
> > +
> > +    @loans.each do |loan|
> > +      self.prepare_event(loan.calendar_start_event)
>
> In Ruby models as well or just in controllers?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/sassafrastech/madeline_system/pull/64/files/3f76163721b12a36c9dd0f22fa6e0bc871fd50c5#r59912789>
>



-- 
Tom Smyth

Worker-Owner, Sassafras Tech Collective
Specializing in innovative, usable tech for social change
sassafras.coop *·* @sassafrastech

Resident, Touchstone Cohousing
touchstonecohousing.org